jenkins-master/jobs/trigger-ondemand/workspace/trigger.py

```diff
@@ -84,7 +84,7 @@ def get_test_packages_url(properties):
 
     revision = properties['revision'][:12]
 
-    client = TreeherderClient(host='treeherder.mozilla.org', protocol='https')
+    client = TreeherderClient(server_url='https://treeherder.mozilla.org')
```

Comment:
The client actually now works without any parameters being passed at all (it will default to Treeherder prod):
https://treeherder.readthedocs.io/rest_api.html#python-client

...though understand if you'd prefer to leave this in :-)